.scalafmt.conf

```diff
@@ -0,0 +1,5 @@
+version = "2.2.1"
```

Comment:
You don't want to add the scalafmt sbt plugin to go with this?

Reply:
Yes. Deleted it temporarily, forgot to add it back.